content/blog/storage-management-with-redfish.md

```diff
@@ -76,15 +84,15 @@ If the value is set to `Disabled`, destructive data actions are allowed. Finally
 
 Imagine you just received a bunch of brand new servers or you want to re-deploy servers for a new project. In this context, you would want to remove completely the entire storage configuration as well as all meta data stored in the physical drives to get a clean Smart Storage subsystem.
 
-Redfish models logical drives as a JSON collection stored in an array of objects represented like: `"LogicalDrives": [{ld-1},{ld-2,...{ld-n}]`. Each `ld-i` of this array contains the necessary properties to describe precisely the corresponding logical drive.
+The HPE OEM storage subsystem models logical drives as a JSON collection stored in an array of objects represented like: `"LogicalDrives": [{ld-1},{ld-2,...{ld-n}]`. Each `ld-i` of this array contains the necessary properties to describe precisely the corresponding logical drive.
 
 The following screenshot shows the `LogicalDrives` array containing one element and its attributes. In it, you can see that this logical drive is a RAID 0 made of a single data drive (`1I:3:1`).
 
 ![Logical drive collection](https://redfish-lab.sourceforge.io/media/redfish-wiki/StorageManagementWithRedfifsh/3-LogicalDriveCollection.png)
 
 ### Removing all logical drives
 
-Removing all the logical drives of a Smart Storage Controller is equivalent to the removal of all the elements of the `LogicalDrives` array. Practically you need to request an empty array to the Redfish server.
+Removing all the logical drives of an HPE SR Smart Storage Controller is equivalent to the removal of all the elements of the `LogicalDrives` array. Practically you need to request an empty array to the Redfish server.
```

Comment:
I apparently missed this earlier as well. The second sentence here isn't clear. I don't know what you mean by "Practically". Should this have been worded as "Basically, you need to request an empty array to the Redfish server."?
Or, maybe, "In practice, you need to request an empty array to the Redfish server."?